src/runtime/basic-runtime-test.js

```diff
@@ -75,11 +75,7 @@ describes.realWin('installBasicRuntime', (env) => {
     expect(progress).to.equal('');
 
     // Install runtime and schedule few more dependencies.
-    try {
-      installBasicRuntime(win);
-    } catch (e) {
-      // Page doesn't have valid subscription and hence this function throws.
```

Comment:
Narrator: It didn't

Reply:
LOL

Reply:
But should it have?

Reply:
Haha, my best guess is no... at least from what's going on in the test and its name